### ISSUE TITLE
Open to migrate to Codeberg?

### ISSUE THREAD

**Issue by aschrijver** [gabek/fedidocs#3]
Hi @gabek 

Yesterday I wrote a SocialHub topic and wiki post on having more cohesion and collab in - what I call - The Grassroots Fediverse. The place where people have their own initiatives, independent and decentralized, yet pursue common dreams and visions for the future of the Fediverse. People who understand that it is the Culture in this grassroots ecosystem that is the great distinguishing factor that makes fedi successful, where so many corporate Social Media fail or aren't truly for-the-people.

As you know, with corporate interests growing, a corporate takeover of both culture and ecosystem looms. FOSS and Free Culture can counter-balance if we manage to get our collab up a notch, and not be a Splinterverse.

Aligning some orgaization structure may be all that's needed. As part of that a logical step is to get a Developer Documentation repository + codeberg pages website going under the Codeberg [Fediverse organization account](https://codeberg.org/fediverse) that SocialHub also uses for the [FEP process](https://codeberg.org/fediverse/fep).

Would you be open to migrate your repo to this codeberg Fediverse org and make your initiative the starting point of this documentation effort. There's Codeberg Pages and CI to replace GH stuff. We will then have codeberg teams allowing multiple people to maintain / be editors, and promote collectively for anyone to contribute their bits and pieces of text.

See: https://socialhub.activitypub.rocks/t/ideating-organization-structure-for-the-grassroots-fediverse-wiki/3037

**Comment by gabek** [comment]
Just to make sure I'm focusing on the right things here, it's not Codeberg explicitly you're asking about, but more about moving to the Fediverse organization who happens to already be using Codeberg.

Who is this organization? What does "under the organization" mean? I know there's a lot of discussion on the forum, but outside of that what is it doing? Who's involved in the organization?

With those questions asked, I'm not opposed to this, but admittedly if the activitypub.rocks organization was actively and successfully involved in these kind of projects I wouldn't have started one in the first place. I'm not trying to be insulting or question all the passion and work you and your organization puts into the Fediverse, but activitypub.rocks was last updated two years ago. So while I don't doubt the goals and ambition, I'm not sure adding more projects under your umbrella that will go untouched is the right move forward.

Again I'm not saying it's a bad idea, but sometimes **decentralizing** things is better, allowing for projects to get done when **centralizing** moves slowly.

**Comment by aschrijver** [comment]
Yes, you are correct. The Fediverse org. I [created this org](https://socialhub.activitypub.rocks/t/social-cg-issue-trackers/1648/2?u=aschrijver) to be on an open platform based on a FOSS code forge (i.e. Codeberg running Gitea then, Forgejo now). It was first used for the purpose of moving the Fediverse Enhancement Proposal process. Before that the FEP was located on Cory Slep's self-hosted forge, with unreliable availability.

The Fediverse org has multiple owners (currently 3) and multiple teams, where one is FEP-editors with 4 members currently. This number can grow, which is good. Idea is to ensure active maintenance and shared privileges (e.g. the W3C repo's on GH have become unmaintained as maintainers are 'AWOL'). Most members are also member at SocialHub, but not all. E.g. one owner is [@lightone](https://mastodon.xyz/@lightone) who created and maintains the popular [Fediverse Party](https://fediverse.party) from there. I co-maintain the fedi-related [Delightful Club](https://delightful.club) curated lists there, which are input to Fediverse Party. The delightful project is part of [Social Coding Movement](https://discuss.coding.social).

So the Fediverse codeberg org is a diverse location for loosely affiliated and otherwise still independent initiatives. And the number of initiatives is likely to grow. Just yesterday on behalf of Helge I create a [fediverse-ideas repository](https://socialhub.activitypub.rocks/t/discuss-the-grassroots-fediverse-organization-structure/3038/8?u=aschrijver).

As for activity of initiatives in general - not just activitypub.rocks as you mention - the "Splinterverse" of overly fragmented initiatives is creating ghost towns all over the web. Some time ago I kept notes on the [substrate of the Fediverse](https://notes.smallcircles.work/_FF2CKvcRy62RrygWkOWHA#) and that doesn't look good. Specifically related to activitypub.rocks I sent an email to @cwebber yesterday, who owns the domain and site, and asked a similar question as I did here.

I imagine we might have a 2-pronged approach to our fedi ecosystem. Two parallel tracks, one for Grassroots evolution (decentralized, as you rightfully say is the better approach. It offers resilience) and the other for Formal steering (whatever W3C activity is ramping up, likely involving the SWICG community group). Here's a depiction how that might look:

![grassroots-fediverse-vs-formal-steering-body](https://user-images.githubusercontent.com/5111931/227706364-00b0d667-27df-42aa-8855-750e99e67b2e.png)

And I stress that on the Grassroots site things are decentralized and independent. Only the minimum level of extra connectivity is provided so as to increase cohesion and collaboration to give the grassroots parallel track sufficient momentum to co-exist with formal steering, and be upheld in the face of corporate takeover forces should that trend indeed progess.

**Comment by wojtekLs** [comment]
@aschrijver I was not going to write here, but after some consideration I chose to write. `Fediverse` (**in the current shape**) is.... well, destined to, quite epic, failure.
Why? Two main reasons:
* wrong people are at the top,
* wrong structure (too many initiatives under one fedi-umbrealla), also core, well, **there is no single place that could be treated as development-home**.
